### ISSUE TITLE
keypoints labeling problem

### ISSUE THREAD

**Issue by LiGuo12** [ryouchinsa/Rectlabel-support#253]
Hi ryo,

We are using your software to complete an annotation on pose estimation. We need to annotate some key points on an image. How can we display multiple key points at the same time? Now, each image only shows one keypoint at a time (the green box in the lower left corner). We are paid users, but we cannot display multiple key points in either rectlabel or rectlabel pro. We need your help. 
![WechatIMG2211](https://github.com/ryouchinsa/Rectlabel-support/assets/119479316/da355758-13e2-4e67-9bd9-088029562a61)

**Comment by LiGuo12** [comment]
Hi Ryo,

Thanks a lot for this answer, and it is really helpful.

We tried using keypoint mode to do annotation, but the connection 
(skeleton) between points was not displayed. How can I make the 
connection between points appear? Will the connection be displayed in 
the final annotation file(coco, csv, yolo formatn)? If we use point mode 
to create annotations, do the pose estimation models of detectron2 and 
yolov8 accept annotations created in point mode? Or are only annotations 
created in keypoint mode accepted by these pre-trained models?

We previously used rectlabel to label object detection and segmentation, 
and exported the labels in coco format, which performed very well on the 
detectron2 pre-trained models. If we use the pretrained pose estimation 
models of detectron2 and yolov8, should we also export to coco format?

By the way, our team uses rectlabel to annotate our data and cite 
rectlabel in 3 of our published papers. We really appreciate the 
software you developed, it works really well.

Best,

Li

**Comment by ryouchinsa** [comment]
Thanks for your detailed feedback.

To add an edge of the skeleton, drag from one to another point holding option button. 
When you right click on the point, edit menu opens.
- “Change keypoint name” to change the keypoint name. 
- “Change keypoint color” to change the keypoint color and the edge color is defined by the source point color.

[Read more](https://rectlabel.com/edit/) about Create keypoints.

![keypoints](https://github.com/ryouchinsa/Rectlabel-support/assets/1954306/f89d1d94-3360-4e0f-8755-0a05a10c68da)

If you are using the Create point mode, you have to convert to the keypoints format to train on Detectron2 and YOLOv8.
If you could export the current annotations in COCO format and send to us support@rectlabel.com, we can code the conversion script using python.

After conversion,  you can import the COCO file to RectLabel, and your annotations will be shown in Create keypoints mode. If you train on Detectron2, please export in COCO format. If you train on YOLOv8, please export in YOLO format.

Thanks for using RectLabel in your team and citing in your papers.
If you could provide the url of each paper, it would be very helpful for us.

**Comment by LiGuo12** [comment]
Hi Ryo,

Thank you for your help.

The title of the published paper is Deep Learning Model Shows Promise 
for Detecting and Grad-ing Sesamoiditis in Horse Radiographs, L. Guo, X. 
Yu, A. Thair, A. Rideout, A. Collins, Z. Jane Wang, M. Hore, American 
Journal of Veterinary Research, 2023 Oct 17, 1(aop), 1-10.DOI: 
https://doi.org/10.2460/ajvr.23.07.0173

Two other papers are still under review.
Thank you for the software, it's very helpful and easy to use.

Best,

Li

**Comment by ryouchinsa** [comment]
Thanks for letting us know about your papers and citing RectLabel.

If you need our support for your keypoints annotations, we are happy to support your team.

We updated our help page so that there are gif movies how to use RectLabel including the skeleton.
https://rectlabel.com/edit/

Best regards,
Ryo

**Comment by ryouchinsa** [comment]
Currently If you need our support to fix this problem, please let us know.
